README.md

```diff
@@ -1,5 +1,11 @@
 # Frictionless CKAN Mapper for JS
 
+![Node.js CI](https://github.com/datopian/frictionless-ckan-mapper-js/workflows/Node.js%20CI/badge.svg)
+
 This is JS version of [Frictionless CKAN Mapper](https://github.com/frictionlessdata/frictionless-ckan-mapper).
 
 A library for mapping CKAN metadata <=> Frictionless metadata.
+
+# Sources
+
+- [Frictionless - Mappings from Other Formats to Frictionless both Profiles and Schemas](https://docs.google.com/spreadsheets/d/1XdqGTFni5Jfs8AMbcbfsP7m11h9mOHS0eDtUZtqGVSg/edit#gid=1925460244)
```

Comment:
@kmanaseryan The code looks good :clap:  but the documentation of the README I think is small. **Note** I don't know if you will improve this in the future, will be?

i.e I'm not up to date with this, so I would like to do:

- How to install
- How to run/use
- How to run tests

Reply:
Yea absolutely agree! I will improve the docs and will add all your points too👍

Reply:
@kmanaseryan sounds good.
I have an example here https://github.com/datopian/ckan-client-js#use if you want to follow someone.

Reply:
@mariorodeghiero I've seen this and actually I added in my list as a nice example of documentation and not only. Very clean and descriptive, thank you!👍